pkg/service/catalog.go

```diff
@@ -260,8 +264,43 @@ func (s *service) GetChildServiceID() string {
 
 // NewPlan initializes and returns a new Plan
 func NewPlan(planProperties *PlanProperties) Plan {
+
+	// commonSchema := &service.ParameterSchemas{
+	// 	ServiceInstances: &service.InstancesSchema{
+	// 		Create: &service.ProvisioningParametersSchema{
+	// 			Parameters: service.GetCommonSchema(),
+	// 		},
+	// 	},
+	// }
+	//ProvisionParamSchema *ParametersSchema      `json:"-"`
+	//UpdateParamSchema    *ParametersSchema      `json:"-"`
+	//BindingParamSchema   *ParametersSchema      `json:"-"`
```

Comment:
Are these deliberately left here for reference? Or were these meant to be removed?